styles.css

```diff
@@ -0,0 +1,76 @@
+* {
+  Box-sizing: border-box; }
+
+a{
+   text-decoration: none;
+}
+a:link {
+  text-decoration: none;
+}
+
+a:visited {
+  text-decoration: none;
+}
+
+a:hover {
+  text-decoration: underline;
+}
+
+
+body { 
+  margin-right: auto;
+  margin-left: auto;
+max-width: 720px;
+}
+
+
+h1,
+h2,
+h3,
+h4,
+a {
+  font-family: "Work Sans", sans-serif;
+}
+h3{
+width:45%;
+  margin-left: 20px;
+}
+div {
+  width:50%;
+border-left: 2px solid pink;
+  padding-left: 10px;
+}
+
+div,h3{
+    display: inline-block;
+  margin-top: 7px;
+  margin-bottom: 7px;
+  vertical-align: top;
+}
```

Comment:
I would recommend using classnames here, instead of base elements. Using the base elements is part of what made it frustrating for you to try to get the layout you wanted on the Exhibitions section.

Here is a video describing in detail how to use class names for this same effect _and_ getting the layout I think you wanted for the Tools section:
https://www.loom.com/share/86e8cdff2c7f4bd59afa7c50cda2acaf

This is the Glitch remix I did as part of that video: https://glitch.com/edit/#!/remix-andrea-p1?path=styles.css%3A61%3A24